docs/introduction/building-and-running.md

```diff
@@ -0,0 +1,43 @@
+# Building and Running a Project
+
+The `subo` command line tool is used again here to build and run your Atmo project.
+
+## Building
+
+Inside the `important-api` directory run: 
+
+```
+subo build .
+```
+
+This automatically compiles each of your Runnables in a Docker container and bundles them together in `runnables.wasm.zip` to be used in Atmo. 
+
+```
+⏩ START: building runnables in .
+ℹ️  🐳 using Docker toolchain
+⏩ START: building runnable: helloworld (rust)
+    Updating crates.io index
+[...]
+
+✅ DONE: bundle was created -> runnables.wasm.zip @ v0.1.0
+```
+
+{% hint style="info" %}
+If you prefer not to use Docker, you can also [build your Runnables locally](https://github.com/suborbital/subo/blob/main/docs/get-started.md#building-without-docker).
+{% endhint %}
+
+## Running a development server
+
+Again, inside the `important-api` directory run: 
```

Comment:
I'd change this line to say 'Now that we have our application bundle built, we can start a development server. In the `important-api` directory, run:'